File: 0657-robot-return-to-origin/0657-robot-return-to-origin.js
var judgeCircle = function(moves) {
    let x = 0;
    let y = 0;

    for (let m of moves) {
        if (m === "U") {
            x += 1;
        } else if (m === "D") {
            x -= 1;
        } else if (m === "R") {
            y += 1;
        } else {
            y -= 1;
        }
    }

    return x === 0 && y === 0;
};

File: 0657-robot-return-to-origin/0657-robot-return-to-origin.py
class Solution(object):
    def judgeCircle(self, moves):
        x=0
        y=0
        for m in moves:
            if m =="U":
                x+=1
            elif m=="D":
                x-=1
            elif m=="R":
                y+=1
            else:
                y-=1
        return x==0 and y==0